extensions/amp-embedly/0.1/amp-embedly.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../src/services';
+import {addParamsToUrl} from '../../../src/url';
+import {dict} from '../../../src/utils/object';
+import {getIframe} from '../../../src/3p-frame';
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {user} from '../../../src/log';
+
+/** @private @const {string} */
+const TAG = 'amp-embedly';
+
+/**
+ * @const {string}
+ */
+const BASE_API_URL = 'https://api.embedly.com/1/oembed?';
+
+/**
+ * Regex used to extract src from returned oEmbed data.
+ * @const {RegExp}
+ * */
+const SRC_REGEXP = /src="([^"]+)"/;
+
+/**
+ * oEmbed resource types.
+ * @const {Object<{PHOTO: string, VIDEO: string, LINK: string, RICH: string}>}
+ */
+const resourceType = Object.freeze({
```

Comment:
It's common style on AMP to define `enums` with `CapitalizedCamelCase`, so please name as `ResourceType`.